src/ejabberd_c2s.erl

```diff
@@ -1057,9 +1058,9 @@ process_outgoing_stanza(Acc, ToJID, <<"presence">>, StateData) ->
     NState;
 process_outgoing_stanza(Acc0, ToJID, <<"iq">>, StateData) ->
     Acc = mongoose_acc:require(xmlns, Acc0),
-    FromJID = mongoose_acc:get(from_jid, Acc),
-    Server = mongoose_acc:get(server, Acc),
-    El = mongoose_acc:get(element, Acc),
+    FromJID = mongoose_acc:get_from_jid(Acc),
+    Server = mongoose_acc:get_server(Acc),
+    El = mongoose_acc:get_element(Acc),
```

Comment:
sometimes I am dreaming about

```erlang
{FromJID, Server, El} = mongoose_acc:get3(from_jid, server, element, Acc)
```

Reply:
ok, I can almost tolerate style like this

```erlang
do_something_cool(Acc) ->
   %% so stupid, nothing is happening here, woohoo!
    FromJID = mongoose_acc:get_from_jid(Acc),
    Server = mongoose_acc:get_server(Acc),
    El = mongoose_acc:get_element(Acc),
    do_something_cool(FromJID, Server, El, Acc).

do_something_cool(FromJID, Server, El, Acc) ->
    %% actually some logic here
    ...
```


Not unpacking Acc in a middle of a function, than packing again and having Acc1, Acc2, .... Acc66 in 500 loc code :D 


TLDR: not related to this PR at all, but acc introduces a lot of boilerplate code.